tgui/packages/tgui/interfaces/OreSilo.tsx

```diff
@@ -222,29 +265,119 @@ const LogsList = (props: LogsListProps) => {
   );
 };
 
-type LogProps = {
-  log: Log;
-};
+const EntryTitle = (log: Log) => {
+  const { action, amount, noun, user_data } = log;
+
+  const Verb = (
+    <Box as="span" className="__actionPart">
+      {action}
+    </Box>
+  );
+
+  const formatAmount = (action: string, amount: number) => {
+    const isSheetAction = action === 'EJECT' || action === 'DEPOSIT';
+    const rawAmount = Math.abs(amount);
+    if (!isSheetAction) {
+      return rawAmount;
+    }
+    const proportionalAmount = rawAmount / 100;
+    return ` ${proportionalAmount} `;
+  };
+
+  const Noun = (
+    <Box as="span" className="__nounPart">
+      {noun}
+    </Box>
+  );
 
-const LogEntry = (props: LogProps) => {
-  const { log } = props;
   return (
-    <Section
-      title={`${capitalize(log.action)}: x${Math.abs(log.amount)} ${log.noun}`}
-    >
-      <LabeledList>
-        <LabeledList.Item label="Time">{log.time}</LabeledList.Item>
-        <LabeledList.Item label="Machine">
-          {capitalize(log.machineName)}
-        </LabeledList.Item>
-        <LabeledList.Item label="Location">{log.areaName}</LabeledList.Item>
-        <LabeledList.Item
-          label="Materials"
-          color={log.amount > 0 ? 'good' : 'bad'}
+    <>
+      {
+        <Box as="span" style={{ textTransform: 'uppercase' }}>
+          {Verb}
+        </Box>
+      }{' '}
+      {formatAmount(action, amount)} {Noun}
+      {', '}
+      {'['}
+      {user_data.Name}
+      {' | '}
+      {
+        <Box
+          as="span"
+          style={{
+            textTransform: 'uppercase',
+          }}
         >
-          {log.rawMaterials}
-        </LabeledList.Item>
-      </LabeledList>
-    </Section>
+          {user_data.Assignment}
+        </Box>
+      }
+      {']'}
+    </>
+  );
+};
+
+const UserItem = (user_data: UserData) => {
+  const {
+    Name,
+    Age,
+    Assignment,
+    'Account ID': accountId,
+    'Account Holder': accountHolder,
+    'Account Assignment': accountAssignment,
+    Accesses,
+    chamelon_override,
+    silicon_override,
+    id_read_failure,
+  } = user_data;
+  const { act, data } = useBackend<Data>();
+  const { banned_users } = data;
+  return (
+    <Stack align="center" className="__UserItem">
+      <Stack.Item>
+        <Box className="__Name">{Name}</Box>
+      </Stack.Item>
+      <Stack.Item>
+        <Box className="__Assignment">{Assignment}</Box>
+      </Stack.Item>
```

Comment:
![image](https://github.com/user-attachments/assets/3110c548-4f27-43c2-bac2-7cab68bce7ab)
Based on how little horizontal margin/spacing exists between these stack items, you might want to consider just a big more horizontal spacing there.